service/src/main/java/bio/terra/pipelines/db/PipelinesDao.java

```diff
@@ -17,11 +21,26 @@ public class PipelinesDao {
               rs.getString("display_name"),
               rs.getString("description"));
 
-  private final JdbcTemplate tpsJdbcTemplate;
+  private final NamedParameterJdbcTemplate jdbcTemplate;
 
   @Autowired
   public PipelinesDao(TspsDatabaseConfiguration tspsDatabaseConfiguration) {
-    this.tpsJdbcTemplate = new JdbcTemplate(tspsDatabaseConfiguration.getDataSource());
+    this.jdbcTemplate = new NamedParameterJdbcTemplate(tspsDatabaseConfiguration.getDataSource());
+  }
+
+  /**
```

Comment:
rather than a getPipeline method, could use a true checkPipelineExists or getPipelineCount method that does essentially a SELECT COUNT(pipeline_id) so that it doesn't pull data out that we're not using, and we can make the error checking more clear at a high level